### DISCUSSION TITLE
Installing into LillyGo T-TWR Plus

### DISCUSSION THREAD

**Discussion by shambahri** [erstec/APRS-ESP#53] [Q&A] [answered]
Hi Team,

I need some help to install into LillyGo T-TWR Plus using Windows.

IF i am using the ESPTool GUI, after flashing to add 0x10000. nothing works. After reseting the LCD is not 
displaying anything.

Please help.

Thanks.
Sam

**Comment by erstec** [comment]
> Hi Team,
> 
> 
> 
> I need some help to install into LillyGo T-TWR Plus using Windows.
> 
> 
> 
> IF i am using the ESPTool GUI, after flashing to add 0x10000. nothing works. After reseting the LCD is not 
> 
> displaying anything.
> 
> 
> 
> Please help.
> 
> 
> 
> Thanks.
> 
> Sam
> 
> 

There is no ESP32S3 MCU support, so T-TWR Plus is not supported

**Reply by shambahri** [reply]
I hope will be support soon, T-TWR Plus has all the HW to support APRS. Thank you again for the info. I will be waiting for this wonderful project. HAM it up. Cheers.

**Reply by erstec** [reply, marked as answer]
It is supported from V1.50

**Reply by shambahri** [reply]
Thank you, works great. 73's
